src/plugins/share/kibana.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "share",
   "version": "kibana",
-  "server": false,
+  "server": true,
```

Comment:
Was this intentional?

Reply:
the plugin contains server part

Reply:
Yes, this means the platform should pick up the `server` directory for server side code. This PR moved the url shortener into the `share` plugin whereas before it was only a client side thing (handling the share registry)